### ISSUE TITLE
Test vizarr

### ISSUE THREAD

**Issue by will-moore** [ome/ome_zarr_test_suite#1]
@joshmoore  I assume we could test vizarr using a Docker container specified in *this* repo instead of in the vizarr repo, in order not to pollute that repo with OME's code?

**Comment by joshmoore** [comment]
Exactly. If there's a Dockerfile somewhere that does what you want, then you can follow https://github.com/ome/ome_zarr_test_suite/blob/main/scripts/omero_ms_zarr_fake . Otherwise, create a subdirectory there that contains a Dockerfile and we'll go from there. :+1:

**Comment by will-moore** [comment]
I was thinking of:

 1) Using Docker with [http-server](https://github.com/http-party/http-server) to serve the test data (with cors).

 2) Then checkout and build a particular branch of vizarr, or possibly use the PR builds e.g. PR 85 [deploy-preview-85](https://deploy-preview-85--vizarr.netlify.app/?source=https://minio-dev.openmicroscopy.org/idr/idr0001-graml-sysgro/pr59_nested/2551.zarr/A/5/0) to consume and display the data.

 3) Finally, use [pageres-cli](https://github.com/sindresorhus/pageres-cli) to grab a browser screenshot of the app. Either this could be compared with a known valid screenshot, or some other simpler validation that at least the image is displayed.

I managed 1) OK, but failed with 3) - getting errors with puppeteer which referred me to https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker but I didn't manage to get any of the solutions there working

**Comment by joshmoore** [comment]
Don't know if it will be of any use but seealso: https://github.com/zarr-developers/zarr_implementations/pull/21
